card/lib/card/query/attributes.rb

```diff
@@ -25,31 +25,31 @@ def right val
       end
 
       def editor_of val
-        acts_alias, actions_alias =
-          "a#{table_id force=true}", "an#{table_id force=true}"
-        joins << Join.new(
+        act_join = Join.new(
           from: self,
-          to: ['card_acts', acts_alias, 'actor_id' ]
+          to: ['card_acts', "a#{table_id force=true}", 'actor_id' ]
         )
-        joins << Join.new(
-          from: ['card_acts', acts_alias],
-          to: ['card_actions', actions_alias, 'card_act_id']
+        joins << act_join
+        action_join = Join.new(
+          from: act_join,
+          to: ['card_actions', "an#{table_id force=true}", 'card_act_id'],
+          superjoin: act_join
         )
-        join_cards val, from_alias: actions_alias, from_field: 'card_id'
+        join_cards val, from: action_join, from_field: 'card_id'
       end
 
       def edited_by val
-        acts_alias, actions_alias =
-          "a#{table_id force=true}", "an#{table_id force=true}"
-        joins << Join.new(
+        action_join = Join.new(
           from: self,
-          to: ['card_actions', actions_alias, 'card_id' ]
+          to: ['card_actions', "an#{table_id force=true}", 'card_id']
```

Comment:
Useless assignment to variable - `force`.<br>Surrounding space missing for operator '='.